### DISCUSSION TITLE
Transmission comment s'assurer que l'appel à notre api fonctionne

### DISCUSSION THREAD

**Discussion by MJMESS** [MTESSDev/FRW#37] [Annonces]
Bonjour, je veux tester l'appel à notre api lors de la transmission. Je ne sais pas comment savoir si l'appel s'effectue, j'ai l'impression que l'appel ne se fait pas. Je n'ai pas d'erreur. Si je clique sur le bouton soumettre, j'ai une confirmation de transmission avec succès. Et dans l'outil avec le bouton "Tester Transmission", même si j'ai supprimé l'option "modeBoutonTesterTransmission: simuler" de mon fichier de transmission, il m'inscrit tout de même que l'appel était simulé selon la configuration.  

![image](https://user-images.githubusercontent.com/122536311/227987321-8992b41e-e6c3-444e-886a-959f312b4729.png)

Est-ce qu'il y a un moyen de vérifier l'appel à notre api lors de la transmission ? 

Merci à l'avance!

**Comment by anisite** [comment]
Bonsoir, vous avez raison, c'est mon erreur, il s'est glissé un bug dans le traitement (en mode simulation seulement) qui affectait le comportement. 

D'ailleurs nos essais de non régressions débutent la semaine prochaine pour valider les changements liés aux outils de dev.

C'est en cours de déploiement, vous pourrez tester dans 15 minutes!
